src/uct/sm/mm/mm_posix.c

```diff
@@ -272,6 +282,22 @@ uct_posix_alloc(uct_md_h md, size_t *length_p, ucs_ternary_value_t hugetlb,
         goto err_free_file;
     }
 
+    /* immediately unlink the file */
+    if (posix_config->use_proc_link == UCS_YES) {
```

Comment:
UCS_YES/UCS_NO is used by "tristate" type, pls treat use_proc_link as regular boolean